Content.Shared/_Goobstation/Clothing/Components/ClothingGrantTagComponent.cs

```diff
@@ -0,0 +1,20 @@
+// SPDX-FileCopyrightText: 2024 Aidenkrz <aiden@djkraz.com>
+// SPDX-FileCopyrightText: 2024 Piras314 <p1r4s@proton.me>
+// SPDX-FileCopyrightText: 2024 VMSolidus <evilexecutive@gmail.com>
+// SPDX-FileCopyrightText: 2025 Aiden <28298836+Aidenkrz@users.noreply.github.com>
+// SPDX-FileCopyrightText: 2025 Misandry <mary@thughunt.ing>
+// SPDX-FileCopyrightText: 2025 gus <august.eymann@gmail.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+namespace Content.Shared._Goobstation.Clothing.Components;
+
+[RegisterComponent]
+public sealed partial class ClothingGrantTagComponent : Component
+{
+    [DataField("tag", required: true), ViewVariables(VVAccess.ReadWrite)]
+    public string Tag = "";
```

Comment:
Please make this into a protoid
```suggestion
    public ProtoId<TagPrototype> Tag = "";
```